orgchart.md

```diff
@@ -76,7 +76,11 @@ SIP is our instant-response team whenever something goes wrong, is about to go w
 
 ### QA
 
-Before we ship a new feature, **Michael Berger** is our resident bug squasher. His job is to break things so we know what needs fixing. He runs rolling tests on all web and mobile versions of Basecamp for quality control. Few things ship without his stamp of quality approval.
+Before we ship a new feature, **Michael Berger** is our resident bug squasher. With the help of some testers at [Aspiritech](https://www.aspiritech.org), he oversees functional testing on new features, and regression tests to make sure existing features aren't adversely affected by new code. Using primarily a black box methodology, we highlight both functional bugs and usability concerns. Our goal is to make sure our products work well across all of the platforms we support, including desktop browsers, desktop apps, mobile browsers, and native iOS and Android apps.
```

Comment:
You can delete the article `a` in _"primarily a black box methodology"_: 

```suggestion
Before we ship a new feature, **Michael Berger** is our resident bug squasher. With the help of some testers at [Aspiritech](https://www.aspiritech.org), he oversees functional testing on new features, and regression tests to make sure existing features aren't adversely affected by new code. Using primarily black box methodology, we highlight both functional bugs and usability concerns. Our goal is to make sure our products work well across all of the platforms we support, including desktop browsers, desktop apps, mobile browsers, and native iOS and Android apps.
```

Also, the second "we" (_"we highlight both…_") is vague. Throughout this doc, "we" usually means "Basecamp." But here a reader might think you mean that "we" is Aspiritech, or perhaps Aspiritech + Michael Berger.

Depending on what you mean, you might want to switch that "we" and the later QA notes to…

- "we at Basecamp"
- "Michael and team"